### PR TITLE
Ot senzing datamodel

### PR DESCRIPTION
**Pull request by oskar700**
This adds the option of downloading Senzing data model in lieu of the default data model.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -134,6 +145,13 @@ AC_ARG_ENABLE([data-download],
                 *) AC_MSG_ERROR([bad value ${enableval} for --disable-data-download]) ;;
               esac], [DOWNLOAD_DATA=true])
 
+AC_ARG_ENABLE([senzing-datamodel],
```
Comment (albarrentine) [inline configure.ac:148]: Can we make it “—model=senzing” and then if there are other models down the road that flag can be a switch?

**Comment by albarrentine** [comment]
Thanks @oskar700, glad the new model training worked out! I downloaded it earlier to do a few spot checks and it looks like it indeed does solve a few issues folks have reported (and probably many more that have not been). Added a few comments on the pull request above. The main initial thought was making it `--model=senzing` which generally allows for models to be swapped out, know there are some language-specific versions out there, etc. Congrats to you and your team and thanks for contributing back!

**Comment by brianmacy** [comment]
Al, working on the differences.  We plan on documenting the delta on our test set per country on our GitHub with any commentary.  I expect we will be doing an updated model every 3-6mo.

File: README.md [hunk 2]
```diff
@@ -175,6 +175,17 @@ If you require a .lib import library to link this to your application. You can g
 lib.exe /def:libpostal.def /out:libpostal.lib /machine:x64
 ```
 
+Installation with an alternative data model
+-------------------------------------------
+
+An alternative data model is available for libposta. It is created by Senzing Inc. for improved parsing on US, UK and Singapore addresses and improved US rural route address handling.
+To enable this add `--enable-senzing-datamodel` to the conigure line during installation:
+```
+./configure --datadir=[...some dir with a few GB of space...] --enable-senzing-datamodel
```
[the remark this answers is not in the document]
Comment (missinglink) [reply README.md:184]: Minor typo 'libposta'

File: configure.ac (hunk 1 above)
Comment (oskar700) [reply configure.ac:148]: I like your suggestion. However, I couldn't find a way to use `—model=` so I went with environment variable: `MODEL=senzing` 
```
./configure --datadir=[path to datadir] MODEL=senzing
```
I'm no autoconfig expert so if you know of a way to use `—model=` then let me know.

File: README.md [hunk 3]
```diff
@@ -175,6 +175,17 @@ If you require a .lib import library to link this to your application. You can g
...
+```
+Further information about this data model can be found at: https://github.com/Senzing/libpostal-data
```
[the remark this answers is not in the document]
Comment (oskar700) [reply README.md:186]: Thanks. Our plan is to add information with address examples and statistics comparisons, broken down by country. This is something I will be working on for the next couple of days.

**Comment by oskar700** [comment]
Thanks @albarrentine for a great review. It is spot on.

File: configure.ac (hunk 1 above)
Comment (albarrentine) [reply configure.ac]: hm yeah have mostly been in Python CLI land recently, guess there's not really a way to add an arbitrary command-line arg with autoconf, but MODEL= does the trick

**Pull request merged**